### PR TITLE
Add gnome-font-viewer

### PR DESCRIPTION
**Pull request by matthiasclasen**
This is 3.28.0. The applied patches are from upstream git.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.gnome.font-viewer

File: org.gnome.font-viewer.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-font-viewer",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=ca.desrt.conf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
Comment (nedrichards) [inline org.gnome.font-viewer.json]: Do you need access to: 
"--filesystem=xdg-run/dconf",
"--filesystem=xdg-config/dconf:ro",

File: org.gnome.font-viewer.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	},
+	{
+	    "name": "gnome-font-viewer",
+	    "buildsystem": "meson",
+	    "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/gnome-font-viewer",
+		    "tag": "3.28.0",
```
Comment (nedrichards) [inline org.gnome.font-viewer.json]: alignment!

Comment (matthiasclasen) [reply org.gnome.font-viewer.json]: Pushed a fix for that

File: org.gnome.font-viewer.json (hunk 1 above)
Comment (matthiasclasen) [reply org.gnome.font-viewer.json]: This one, I am curious about - under Wayland, GTK+ directly reads GSettings (the ones that back  GtkSettings). So, while the font viewer does not itself use any settings, it still needs access. 

We clearly need a better story for dconf.

Comment (nedrichards) [reply org.gnome.font-viewer.json]: Agree. Those four lines are the canonical examples from the docs though: http://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access but yeah, there's a reason why this upstream ticket has such a low number... https://github.com/flatpak/flatpak/issues/78

**Comment by nedrichards** [comment]
bot, build org.gnome.font-viewer

**Comment by matthiasclasen** [comment]
anything left to do here ?

File: org.gnome.font-viewer.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-font-viewer",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=ca.desrt.conf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
Comment (TingPing) [inline org.gnome.font-viewer.json]: Missing `--filesystem=xdg-run/dconf --filesystem=~/.config/dconf:ro` ?

File: org.gnome.font-viewer.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                    "tag": "3.28.2",
+                    "commit": "5c6d3bf91ef2bec24e51ed80d32a7c8155116aa1"
+                }
+            ]
+	},
+	{
+	    "name": "gnome-font-viewer",
+	    "buildsystem": "meson",
+	    "builddir": true,
+            "sources": [
```
Comment (TingPing) [inline org.gnome.font-viewer.json]: Whitespace.

**Comment by nedrichards** [comment]
bot, build org.gnome.font-viewer

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.font-viewer

**Comment by nedrichards** [comment]
@matthiasclasen anyone else from upstream to add to the repo?

**Comment by matthiasclasen** [comment]
You could add Cosimo, for old-times sake...

**Pull request closed without merging**